pkg/logs/config/integration_config.go

```diff
@@ -139,3 +139,20 @@ func (c *LogsConfig) Compile() error {
 	}
 	return nil
 }
+
+func (c *LogsConfig) updateTags(tags []string) {
+	tagSet := make(map[string]struct{})
+
+	for _, tag := range tags {
+		tagSet[tag] = struct{}{}
+	}
+	for _, tag := range c.Tags {
```

Comment:
does the config always override run time elements in the other part of the code ?